packages/recommend/src/methods/getLookingSimilar.ts

```diff
@@ -0,0 +1,18 @@
+import { BaseRecommendClient, LookingSimilarQuery, WithRecommendMethods } from '../types';
+import { getRecommendations } from './getRecommendations';
+
+type GetLookingSimilar = (
+  base: BaseRecommendClient
+) => WithRecommendMethods<BaseRecommendClient>['getRelatedProducts'];
```

Comment:
Might not be related to the failing tests, but should this be like this instead?

```suggestion
) => WithRecommendMethods<BaseRecommendClient>['getLookingSimilar'];
```

Reply:
Good catch!